### ISSUE TITLE
image v0.8.11 not published

### ISSUE THREAD

**Issue by JBOClara** [kubernetes/node-problem-detector#692]
FYI, the latest image is not published

```
gcloud container images list-tags k8s.gcr.io/node-problem-detector/node-problem-detector
DIGEST        TAGS          TIMESTAMP
f8f66daddc18  v0.8.10       2021-09-01T01:03:18
c4a75e50c3ee  v0.8.9        2021-06-25T22:52:08
072be0789048  v0.8.8        2021-05-15T03:30:50
0ce71ef6d759  v0.8.7        2021-02-19T02:17:37
22ac8343195e  v0.8.6        2021-01-22T22:44:17
07645e262314  v0.8.5        2020-11-19T01:27:27
3b5f074cf18a  v0.8.4        2020-09-01T07:14:34
9343960c22c6  0.8.3,v0.8.3  2020-07-30T20:09:31
```

**Comment by ialidzhikov** [comment]
@vteratipally if you are working on it, can you assign it to yourself? (ref https://github.com/kubernetes/node-problem-detector/pull/684#issuecomment-1206404338)

**Comment by dfateyev** [comment]
Please consider to push v0.8.11 tag to GCR. Its absence affects all default installations via Helm.

**Comment by maxres-ch** [comment]
Hi we're also having issues with the fact that the image is missing.
```
❯ docker pull k8s.gcr.io/node-problem-detector/node-problem-detector:v0.8.11
Error response from daemon: manifest for k8s.gcr.io/node-problem-detector/node-problem-detector:v0.8.11 not found: manifest unknown: Failed to fetch "v0.8.11" from request "/v2/node-problem-detector/node-problem-detector/manifests/v0.8.11".```

**Comment by dlovelace10** [comment]
Please publish image node-problem-detector:v0.8.11. We downgraded to node-problem-detector:v0.8.10 and it solved the issue.

**Comment by vteratipally** [comment]
Released v0.8.12

Docker Image: k8s.gcr.io/node-problem-detector/node-problem-detector:v0.8.12

/close
